Final Legal Chatbot Files/chainlit_ui.py

```
@@ -0,0 +1,485 @@
+                            },
+                            {"type": "input_image", "image_url": data_url},
+                        ],
+                    }
+                ],
+            )
+            page_text = (resp.output_text or "").strip()
+            if page_text:
+                extracted_pages.append(page_text)
+
+        return "\n\n".join(extracted_pages).strip(), ""
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)